_utility/send_gh_comment.sh

```diff
@@ -1,34 +1,43 @@
 #!/bin/bash
-set -e
+set +e
 
-if [ "$#" -ne 5 ]; then
-  echo "Insufficient arguments, see usage and example for further assistance."
-  echo -e "Usage: $0 REPO_OWNER REPO_NAME PR_NUMBER AUTH_TOKEN DEPLOYMENT_URL\n"
-  echo "Example: $0 codurance site 4 token https://bucket-name.amazon.com/index.html"
-  exit 1
+REPO_OWNER=''
+REPO_NAME=''
+PR_NUMBER=''
+GITHUB_TOKEN=''
+DEPLOYMENT_URL=''
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
would a combination of ' and " on this line work - would make it easier to read and understand whats happening